src/infinite-list/index.html

```diff
@@ -1,17 +1,17 @@
 <ul ref="wrapper" class="gb-list__wrapper {isGrid ? 'gb-list__wrapper--grid' : 'gb-list__wrapper--column'}">
-  <li data-is="gb-list-item" class="load first-loader" if="{$infinite.isFetchingBackward}" _consumes="infinite">
-    <gb-infinite-loader label="{$infinite.loaderLabel}"></gb-infinite-loader>
+  <li data-is="gb-list-item" class="load first-loader" _consumes="infiniteList" if="{props.isFetchingBackward}" _props="{parent.childProps()}">
```

Comment:
should be just `childProps()` because this is in the same context as infinite-list